src/swtpm/Makefile.am

```diff
@@ -117,7 +117,7 @@ swtpm_cuse_CFLAGS = \
 	$(HARDENING_CFLAGS)
 
 swtpm_cuse_LDADD = \
-	-L$(PWD)/.libs -lswtpm_libtpms \
+	-L$(PWD)/.libs -lswtpm_libtpms -ltpms \
```

Comment:
Same here. Does the configure script report a warning regarding not being able to determine the crypto library used in libtpms? This would mean this it cannot find the libtpms.pc file. Also, does the following work on your system?

```
$ pkg-config --libs libtpms
-ltpms
```

If not, then libtpms got installed in the wrong directory where pkg-config does not search for .pc files. You may want to add something like `--libdir=/usr/lib64` or `--libdir=/usr/lib/x86_64-linux-gnu` to the configure line of libtpms so the libtpms.pc gets installed in the proper directory. On Fedora for example the .pc files are all in /usr/lib64/pkgconfig and therefore libtpms.pc needs to go there as well.

Reply:
Yes, `pkg-config --libs libtpms` does not find libtpms. This problem can be solved by adding `PKG_CONFIG_PATH`, I think this should make it fail in the configure phase, not in the final compilation.

Reply:
I fixed this now in PR #286. This should solve the issue, but have a look, please.